llvm/lib/MC/MCELFStreamer.cpp

```diff
@@ -699,17 +699,16 @@ void MCELFStreamer::setAttributeItems(unsigned Attribute, unsigned IntValue,
 
 MCELFStreamer::AttributeItem *
 MCELFStreamer::getAttributeItem(unsigned Attribute) {
-  for (size_t I = 0; I < Contents.size(); ++I)
-    if (Contents[I].Tag == Attribute)
-      return &Contents[I];
+  for (AttributeItem &Item : Contents)
```

Comment:
Could use llvm::find_if here, but might not be better - up to you.

Reply:
Using `llvm::find_if` requires a comparison with `Contents.end()` here and might be longer.
The for each loop LGTM.

Reply:
Thank you for the review!  We could reduce the line count by one, but the simple "for" loop might be more readable here:

```
  auto It = llvm::find_if(
      Contents, [&](AttributeItem &Item) { return Item.Tag == Attribute; });
  return It != Contents.end() ? &*It : nullptr;
```